weld/parser.rs

```diff
@@ -485,8 +487,115 @@ impl<'t> Parser<'t> {
         Ok(cast_expr)
     }
 
+    /// Parses annotations in the format "@(<annotation name>: <annotation value>,...)".
+    fn parse_annotations(&mut self, annotations: &mut Annotations) -> WeldResult<()> {
```

Comment:
Can you add these to `language.md` describing the syntax, valid annotations, etc.